timev.rb

```diff
@@ -1,112 +1,90 @@
-#
-# call-seq:
-#    Time.now -> time
-#
-# Creates a new Time object for the current time.
+# Creates a new \Time object from the current system time.
 # This is the same as Time.new without arguments.
 #
-#    Time.now            #=> 2009-06-24 12:39:54 +0900
+#    Time.now # => 2009-06-24 12:39:54 +0900
 def Time.now(in: nil)
   new(in: __builtin.arg!(:in))
 end
 
+# _Time_
+#
+# This form accepts a \Time object +time+
+# and optional keyword argument +in+:
+#
+#   Time.at(Time.new)               # => 2021-04-26 08:52:31.6023486 -0500
+#   Time.at(Time.new, in: '+09:00') # => 2021-04-26 22:52:32.1480341 +0900
+#
+# _Seconds_
+#
+# This form accepts a numeric number of seconds +sec+
+# and optional keyword argument +in+:
 #
-# call-seq:
-#    Time.at(time) -> time
-#    Time.at(seconds_with_frac) -> time
-#    Time.at(seconds, microseconds_with_frac) -> time
-#    Time.at(seconds, milliseconds, :millisecond) -> time
-#    Time.at(seconds, microseconds, :usec) -> time
-#    Time.at(seconds, microseconds, :microsecond) -> time
-#    Time.at(seconds, nanoseconds, :nsec) -> time
-#    Time.at(seconds, nanoseconds, :nanosecond) -> time
-#    Time.at(time, in: tz) -> time
-#    Time.at(seconds_with_frac, in: tz) -> time
-#    Time.at(seconds, microseconds_with_frac, in: tz) -> time
-#    Time.at(seconds, milliseconds, :millisecond, in: tz) -> time
-#    Time.at(seconds, microseconds, :usec, in: tz) -> time
-#    Time.at(seconds, microseconds, :microsecond, in: tz) -> time
-#    Time.at(seconds, nanoseconds, :nsec, in: tz) -> time
-#    Time.at(seconds, nanoseconds, :nanosecond, in: tz) -> time
+#   Time.at(946702800)               # => 1999-12-31 23:00:00 -0600
+#   Time.at(946702800, in: '+09:00') # => 2000-01-01 14:00:00 +0900
 #
-# Creates a new Time object with the value given by +time+,
-# the given number of +seconds_with_frac+, or
-# +seconds+ and +microseconds_with_frac+ since the Epoch.
-# +seconds_with_frac+ and +microseconds_with_frac+
-# can be an Integer, Float, Rational, or other Numeric.
+# <em>Seconds with Subseconds and Units</em>
 #
-# If +in+ argument is given, the result is in that timezone or UTC offset, or
-# if a numeric argument is given, the result is in local time.
-# The +in+ argument accepts the same types of arguments as +tz+ argument of
-# Time.new: string, number of seconds, or a timezone object.
+# This form accepts an integer number of seconds +sec_i+,
+# a numeric number of milliseconds +msec+,
+# a symbol argument +:millisecond+,
```

Comment:
I would replace this line with `a symbol argument for the subsecond unit type (defaulting to :usec)`.  Then I would remove the two paragraphs below, keeping only the examples.

Reply:
Done.